services/booking/search.rb

```diff
@@ -0,0 +1,26 @@
+class BookingService
+  class Search < ApplicationService
+    def initialize(params)
+      @route_id = params[:route_id]
+    end
+
+    def call
+      booked_seats
```

Comment:
What's the point of having a method that just calls another method?
You could've used `alias` or `alias_method` in Rails but there's no such thing in Sinatra